recipe/build.sh

```diff
@@ -1,7 +1,19 @@
 mkdir build
 cd build
 
-cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+
+if [ $(uname) == Linux ]; then
```

Comment:
These changes could be moved to https://github.com/conda-forge/toolchain-feedstock/blob/master/recipe/activate.sh